cbdefense.json

```diff
@@ -1137,8 +1138,9 @@
                     "order": 2
                 },
                 "json_fields": {
-                    "description": "Other configuration fields in JSON format",
+                    "description": "Other configuration fields in JSON format. Defaults to '{\"sensorSettings\": []}' if left empty",
```

Comment:
`Defaults to '{\"sensorSettings\": []}' if left empty",` move this too verbose and add something like that "If we send blank fields, the action returns failed status and creates policy to avoid this scenario"